reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -211,7 +211,7 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
             it.entries.associateTo(metadata) { (key, value) -> key.toString() to value.toString() }
         }
 
-        val extraColumns = (scanRecord.data["excel_report_extra_columns"] as? List<*>)?.let { extraColumns ->
+        val extraColumns = (ortResult.data["excel_report_extra_columns"] as? List<*>)?.let { extraColumns ->
```

Comment:
Commit message: Better say something like "Read extra columns from OrtResult instead of ScanRecord" to simply describe the change. How we use the data maps internally should not be relevant here, and lacks context.

Reply:
Well, strictly speaking I'm not saying *how* we use the maps internally, but I'm just referring to the names of two maps that we already read out a few lines above my change. I wanted to emphasize these already use the `ortResult` variable, and we should do so here, too.

Reply:
Thanks, I wasn't aware of that. I don't like that we expect those key names here, we should replace that with a more generic approach at some time.